examples/tutorials/data/magic.py

```diff
@@ -360,13 +367,13 @@
 )
 dataset_simulated.peek()
 plt.show()
+# sphinx_gallery_thumbnail_number = 5
```

Comment:
To align it with other instruments thumbnails, I would select the peek plot also for MAGIC.